[2023-04-21 09:15:00,287][root][INFO] - 
Overwritten parameters in the config: 
mode=tiling
[2023-04-21 09:15:00,287][root][INFO] - 
---------------------------------------
Let's start tiling for segmentation !!!
---------------------------------------
[2023-04-21 09:15:00,511][matplotlib][DEBUG] - matplotlib data path: C:\Users\abfernan\.conda\envs\geo_deep_env\lib\site-packages\matplotlib\mpl-data
[2023-04-21 09:15:00,519][matplotlib][DEBUG] - CONFIGDIR=C:\Users\abfernan\.matplotlib
[2023-04-21 09:15:00,519][matplotlib][DEBUG] - interactive is False
[2023-04-21 09:15:00,519][matplotlib][DEBUG] - platform is win32
[2023-04-21 09:15:00,583][matplotlib][DEBUG] - CACHEDIR=C:\Users\abfernan\.matplotlib
[2023-04-21 09:15:00,583][matplotlib.font_manager][DEBUG] - Using fontManager instance from C:\Users\abfernan\.matplotlib\fontlist-v330.json
[2023-04-21 09:15:01,007][fiona.session][DEBUG] - Could not import boto3, continuing with reduced functionality.
[2023-04-21 09:15:01,015][fiona._env][DEBUG] - GDAL_DATA found in environment.
[2023-04-21 09:15:01,023][fiona._env][DEBUG] - PROJ_DATA found in environment.
[2023-04-21 09:15:02,655][pyproj][DEBUG] - PROJ_ERROR: Cannot open https://cdn.proj.org/us_noaa_mnhpgn.tif: schannel: next InitializeSecurityContext failed: Unknown error (0x80092012) - The revocation function was unable to check revocation for the certificate.
[2023-04-21 09:15:02,783][pyproj][DEBUG] - PROJ_ERROR: Cannot open https://cdn.proj.org/us_noaa_nbhpgn.tif: schannel: next InitializeSecurityContext failed: Unknown error (0x80092012) - The revocation function was unable to check revocation for the certificate.
[2023-04-21 09:15:02,975][pyproj][DEBUG] - PROJ_ERROR: Cannot open https://cdn.proj.org/us_noaa_emhpgn.tif: schannel: next InitializeSecurityContext failed: Unknown error (0x80092012) - The revocation function was unable to check revocation for the certificate.
[2023-04-21 09:15:03,151][pyproj][DEBUG] - PROJ_ERROR: Cannot open https://cdn.proj.org/us_noaa_uthpgn.tif: schannel: next InitializeSecurityContext failed: Unknown error (0x80092012) - The revocation function was unable to check revocation for the certificate.
[2023-04-21 09:15:03,831][root][ERROR] - The key download_data as a None value.
[2023-04-21 09:15:03,831][root][ERROR] - The key multiprocessing as a None value.
[2023-04-21 09:15:03,831][root][ERROR] - The key write_dest_raster as a None value.
[2023-04-21 09:15:03,831][root][ERROR] - The key clahe_clip_limit as a None value.
[2023-04-21 09:15:04,071][tiling_segmentation][WARNING] - Data path exists: C:\Users\abfernan\CrossCanFloodMapping\DatasetCNNFlood\Class1RawData.csv\template_project. Make sure patches belong to the same experiment.
[2023-04-21 09:15:04,071][tiling_segmentation][WARNING] - Debug mode activated. Some debug features may mobilize extra disk space and cause delays in execution.
